## Pathogenesis

### Mechanism of Host Infection by SARS-CoV-2

This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.

#### Primary Transmission and Viral Entry

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
The ACE2 receptor is expressed in numerous organs, such as the heart, kidney, and intestine, but most prominently in alveolar epithelial cells, which is expected to contribute to the virus' association with lung pathology [@doi:10.1101/2020.02.24.963348; @doi:10.1007/s00134-020-05985-9].
The S protein is a highly glycosylated trimer that requires two proteolytic cleavage events, leading to substantial conformational changes, to achieve viral fusion with the host cell membrane [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
Each protomer is composed of an S1 and an S2 subunit, which mediate receptor binding and viral fusion, respectively.
The priming proteolytic events occur sequentially, first at the S1/S2 junction and then at the S2’ site, ultimately resulting in the shedding of the S1 subunit and transitioning of the S2 subunit to a more stable, fusion-conducive conformation.

Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein [@doi:10.1016/j.cell.2020.02.052].
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9].
Interestingly, SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7].
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716].
Differences in S protein sequence between SARS-CoV and SARS-CoV-2 may also partially account for the increased transmissibility seen in the current COVID-19 pandemic.
Recent studies have reported conflicting binding constants for the S protein-hACE2 interaction, though they agree that SARS-CoV-2 S protein binds with equal if not greater affinity than does SARS-CoV S protein [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor, and the crystal structure of the C-terminal domain of SARS-CoV-2 S protein in complex with human ACE2 reveals stronger interaction and higher affinity for receptor binding than SARS-CoV [doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV S protein, 8 are conserved in SARS-CoV-2 and the remaining 6 are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].
Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach. Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].
Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry in vitro, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052].

#### Viral Replication and Spreading

- Basic introduction into replication cycle
- What are the routes of transmission

##### Mechanism of viral replication within cells

##### Mechanism of viral spreading to neighbor cells

##### Factors enhancing viral spreading

Viral progression may be enhanced by active upregulation of ACE2 on cell surfaces following or during a response to infection.
In several preliminary assays and an analysis of previous microarray data, Wang et al. reported that ACE2 is significantly upregulated following infection by other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in the upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, which may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
[@doi:10.1016/j.immuni.2020.06.017] describe the concept of the cytokine storm in detail, although [@doi:10.1001/jamainternmed.2020.3313] claim that the widespread use of the term can be misleading.

##### Person-to-person transmission

#### Reproduction Number and Dynamics of Transmission

Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@doi:10.1126/science.abb5659; @isbn:9780691116174].
A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].

Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. The different estimates of R~0~ should not be interpreted as a range of estimates of the same underlying parameter.
Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R~0~.
For data-derived estimates, in one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].
Using the same model as for the French population, this study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~=2.79.

Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R~t~=2.38, and decreasing to R~t~ = 1.34 (using data from Jan 24 to Feb 3) or R~t~=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221].
In South Korea, R~t~ was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@tag:Park2020_distancing].
Metro data was also analyzed to estimate the effects of social distancing measures.
R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R~t~ remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported [@doi:10.1101/2020.03.20.20038422].
In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.

More generally, population-level epidemic dynamics can be both observed and modelled [@doi:10.1126/science.abb5659].
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs [@doi:10.1073/pnas.2004064117] or the comparison of predicted outcomes among prevention and control strategies [@doi:10/ggp3xq; @doi:10.1073/pnas.2004978117].
Many current efforts to model R~t~ have led to tools that assist the visualization of estimates in real time (or over recent intervals) [@url:https://epiforecasts.io/covid; @url:https://rt.live].
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.



### Immune Response to SARS-CoV-2

- Cellular responses to SARS-CoV-2 infection
- What is causing neutropenia and lymphopenia observed in COVID-19 patients
- Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover
- Cytokines and other soluble factors contribution to immune response

### Systems-level approaches for understanding SARS-CoV-2 pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7] or from BALF, throat swabs, or isolates of the virus cultured from BALF [@doi:10/ggjr43].
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43].
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].

The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis.
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507].
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.1038/s41586-020-2332-7].
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients might show some efficacy to cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
Using such databases, the systems level reconstruction of the PPI (Protein-Protein Interaction) enabled the generation of hypotheses on the mechanism of action of SARS-CoV-2 and suggested drug targets.

#### Protein-protein interaction networks

In an initial study [@doi:10.1101/2020.03.22.002386], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins that interact with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
The ranking of pathogens with respect to their interactome's similarity to SARS-CoV-2 suggested *West Nile Virus*, *Mycobacterium tuberculosis*, and *Human papillomavirus* as the top three hits.
However, given the lung symptoms associated with COVID-19, the *Mycobacterium tuberculosis* host-pathogen interactome could also provide new insights to the mechanism of SARS-CoV-2 infection.
In addition, it was suggested that the envelope protein E could disrupt the host bromodomain-containing proteins, i.e., BRD2 and BRD4, binding to histone.
The Spike protein S could likely intervene in the virus fusion through modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation.

Another study [@doi:10.1101/2020.03.31.019216], used patient-derived peripheral blood mononuclear cells (PBMCs) to identify 251 host proteins targeted by SARS-CoV-2 and the disruption of more than 200 host proteins following the infection.
The network analysis showed in particular that non-structural proteins 9 and 10 (nsp9 and nsp10) interacted with NKRF that usually represses NFKB.
This finding could explain the exacerbation of the immune response that shape the pathology and the high cytokine levels possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that protein E of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, that could inhibit anti-apoptosis proteins BCL2 and trigger the apoptosis of T cells.
Several known compounds were suggested to disrupt the host-pathogen protein interactome were suggested mostly through the inhibition of host proteins.

#### Transcriptomics of COVID-19

To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection. 
Although we make an effort to evaluate all relevant studies, we realize that this area is rapidly evolving.

Two studies profiled expression after SARS-CoV-2 infection in multiple human cell lines (Figure {@fig:cell-lines-moi}A) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV). 
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with adenovirus (AdV)-based vectors expressing mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2). 
All infections were performed at a high multiplicity of infections (MOI, 2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
Additionally, they measured transcriptional responses to IAV in NHBE cells and nasal washes of ferrets.
Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
In general, the interferons (IFNs) activate intracellular antimicrobial programs and are the major first line of defense against viruses [@doi:10.1016/j.chom.2020.05.008].
IFNs are a type of cytokines along with chemokines, interleukins (IL), lymphokines and tumor necrosis factor (TNF).
Cytokines can be classified based on the nature of the immune response or based on their specific roles, as reviewed by [@doi:10.1016/j.bbamcr.2014.05.014]).
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under both high- and low-MOI conditions. 
The cytokines IL-6 and IL1RA were uniquely elevated in response to SARS-CoV-2.
However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but did in cells at high MOI.
This finding suggests that IFN induction is MOI dependent. 
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III. 
Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response. 
However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene, which is considerably longer.

A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line). 
They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV at different hpi (4-36 hours) with an MOI of 0.3. 
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads), suggesting a role of cell-type specific host factors.
Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct responses associated with different hosts following infection. 
In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of ISGs. 
Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935]. 
We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available analyzed data from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
In general, we found high correlations among the same host irrespective of the study and the virus infected, suggesting that the response is host-specific. 
For instance, Calu-3 cells infected with SARS-CoV by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).
Moreover, COVID-19 patients had the lowest correlations among all the cells used, with the highest value being 0.29.
This discrepancy suggests that the _in vitro_ models used may not necessarily imitate the human response, underscoring the importance of follow-up in additional models.

![**Profiling and correlation of expression response after respiratory viruses infection in various hosts.** (A) Overview of hosts and respiratory viruses used in two studies [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. MOI:Multiplicity of infections; PFU:Plaque-forming unit; a:ACE2-expressing A549 cells; b:Data used from GSE56192. (B) Pearson's correlation of expression response (Log2 fold change) between various hosts. Note that we used the data from bulk RNA-seq produced after the infection of unmodified viruses.](images/cell-lines-moi.jpg){#fig:cell-lines-moi secno=1}

<!-- Start with scRNA-seq studies in healthy tissues (may be put this before Blanco-Melo et al), then summarize scRNA-seq studies in COVID-19 patients] -->
