## Diagnostics

Identifying individuals who have contracted COVID-19 is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea's successful strategy for controlling the spread of the virus.
Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
A range of diagnostic approaches from a methodological standpoint are being or could possibly be developed.

There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Molecular tests are essential for identifying individuals for treatment and alerting their contacts to quarantine and be alert for possible symptoms.
While serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past based on disease symptoms, their potential for false positives means that they are not currently recommended for this use.
However, serological tests are critical at the population level from an epidemiological perspective, as they can be used to estimate the extent of the infection in a given area.
Thus, they may be used to better understand the percent of infected cases that develop severe disease as well as to guide public health and economic decisions regarding resource allocation and counter-disease measures.

### Molecular Tests

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
This first requires identifying biospecimens that are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sources for a sample used in a molecular test include nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx], or stool samples [@doi:10.1002/jmv.25742].
Given a sample from a patient, molecular tests involve a number of steps to analyze a sample and produce results.
When testing for RNA viruses like SARS-CoV-2, pre-processing is needed in order to create DNA, which can then be replicated during PCR, from the initial RNA sample.
The DNA can then be amplified with PCR.
Some tests use the results of the PCR to determine presence or absence of the pathogen, but in other cases, it may be necessary to sequence the amplified DNA.
Sequencing requires an additional pre-processing step: library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters [@doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding "barcoding" to identify a particular sample in the sequence data, which is useful for pooling samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
Sequential pattern matching is then used to identify unique subsequences of the virus that identify it in specific.
If sufficient subsequences are found, the test is considered positive.

#### RT-PCR

Real-Time Polymerase Chain Reaction (RT-PCR) tests determine whether a target is present by measuring the rate of amplification during PCR compared to a standard.
When the target is RNA, such as in the case of RNA viruses, the RNA must be converted into complementary DNA during pre-processing.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
This test uses RT-PCR with reverse transcription [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the _ORF1b_ of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (_E_), and the Nucleocapsid protein gene (_N_).
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay did not give any false positive results.

#### qRT-PCR

Chinese researchers developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two gene regions of the viral genome, _ORF1b_ and _N_ [@doi:10.1093/clinchem/hvaa029].
Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the _N_ gene to screen patients, while the _ORF1b_ gene region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.

##### dPCR

Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR. 
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This leads to a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples--43 samples from suspected positive patients, and 14 from supposed convalescents--that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR. 
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR [@doi:10.1080/22221751.2020.1772678].
While not a complete head-to-head comparison to RT-PCR in all aspects-- e.g. no samples testing positive using RT-PCR were evaluated by ddPCR-- the study shows the potential of dPCR for viral detection in highly diluted samples.
In a second study, Dong et al. [@doi:10.1101/2020.03.14.20036129] compared the results of qRT-PCR and ddPCR testing for SARS-CoV-2 in 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
Of the qRT-PCR negative or suspected samples, a total of 61 (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
Of 75 patient samples from contacts and close contacts, 48 were tested negative with both methods and patients indeed remained healthy.
Within the remaining 27 patient samples, 10 tested positive, 1 negative, 16 suspect by qRT-PCR.
15 out of 16 suspect samples and the negative test result were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
Importantly, all samples that tested positive using qRT-PCR also tested positive using ddPCR.
Among the 16 convalescent patients, qRT-PCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that dPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.

#### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1101/2020.03.26.20039438], who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.


##### CRISPR-based detection

Two American companies, Mammoth Biosciences and Sherlock Biosciences, adapted their CRISPR-based detection technology [@doi:10.1016/j.cell.2014.05.010] for COVID-19 diagnostics to increase testing throughput and accessibility [@doi:10.1101/2020.03.06.20032334].
Their methodology involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, a rapid, isothermal nucleic acid amplification technique, and application of their CRISPR-Cas12-based technology.
In the assay designed by Mammoth Biosciences, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N, E and RdRP regions.
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection [@doi:10.1101/2020.03.06.20032334].
This assay has been reported to have a sensitivity as high as detection of 70-300 copies of the target RNA/µl, requires simple, accessible equipment, and can output an easily-interpretable result in approximately 30 minutes [@doi:10.1126/science.abb8400].
Initial testing with patient samples (n = 23) demonstrated a positive predictive value of 100% and a negative predictive value of 91.7%, which is highly competitive with the CDC's current testing standard using qRT-PCR.
This test seeks to develop a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.

#### Limitation of Molecular Tests

Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:

1. False-negative responses, which can present a significant problem to large-scale testing.
   To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
2. Uncertainty surrounding the SARS-CoV-2 viral shedding kinetics, which could affect the result of a test depending on when it was taken [@doi:10.1038/d41587-020-00002-2].
3. Type of specimen, as it is not clear which clinical samples are best to detect the virus [@doi:10.1038/d41587-020-00002-2].
4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
5. Timing of the test, which might take up to 4 days to give results [@doi:10.1126/science.abb8400].
6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
7. Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.

Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].

### Serological Tests

Although diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the extent of the disease over time.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Additionally, they can help scientists to understand why the disease has a different course among patients, as well as what strategy might work to manage the spread of the infection.
Furthermore, serological tests hold significant interest at present because they can provide information relevant to advancing economic recovery and allowing reopenings.
For instance, people that have developed antibodies can plausibly return to work prior to the others, based on (still-unproven) protective immunity [@doi:10.1101/2020.04.14.20065771]. 
On a related note, for some infectious agents an epidemic can be stopped from growing through herd immunity in which enough of the population is immune to infection through vaccination and/or prior experience with infection.
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible [@isbn:9780199209996].
However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region deaths and accumulated cases with the estimated factor of undetected cases, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.

#### Current Approaches

Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions [@url:https://www.fda.gov/media/136625/download].
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma of patients (blood sample) suspected to have developed the infection [@url:https://www.fda.gov/media/136625/download].
Such tests allow for the progress of the viral disease to be understood, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay results can be read within 15-20 minutes [@url:https://www.fda.gov/media/136625/download].
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].

#### Limitations of Serological Tests

Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.

### Possible Alternatives to Current Practices for Identifying Active Cases

Clinical symptoms are too similar to other types of pneumonia to be sufficient as a sole diagnostics criterion.
In addition, as noted above, identifying asymptomatic cases is critical.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% [@doi:10.2807/1560-7917.ES.2020.25.16.2000508].
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.

X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

### Challenges to Diagnostic Approaches

#### Limitations to Implementation of Large-Scale Testing
More information to follow.

#### Strategies and Considerations for Determining Whom to Test

Currently, Coronavirus tests are limited to people that are in danger of serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
Specifically, the individuals at risk include:

- people with severe symptoms
- people showing mild symptoms that have been in contact with a person who has developed the infection
- people with underlying health conditions


However, this method of testing administration does not detect a high proportion of infections and does not allow for test-and-trace methods to be used.
Individuals who are asymptomatic (i.e. potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
For instance, a recent study from Imperial College estimates that in Italy the true number of infections is around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million, corresponding to ~1.5% of the population, but 12% of individuals tested statewide and up to ~20% in some places tested positive when evaluated with antibody tests [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
